config/default.js

```diff
@@ -124,6 +124,11 @@ module.exports = {
       save: 'save',
       update: 'update',
     },
+    setup: {
+      path: '/create-admin',
```

Comment:
I'm not a fan of having this set in settings. This should be internal to Punchcard. The actions we do this with for internationalization, and that'll be going away once we have i18n in place. Please change this to be a Punchcard internal URL. Too much can go wrong with allowing a user to set the path directly.